src/AWS.jl

```diff
@@ -473,19 +471,31 @@ function (service::QueryService)(aws::AWS.AWSConfig, operation::String, args::Ab
 end
 (service::QueryService)(operation::String, args::AbstractDict{String, <:Any}=Dict{String, Any}()) = service(AWSConfig(), operation, args)
 
-function (service::JSONService)(aws::AWS.AWSConfig, operation, args=[])
-    return AWSCore.service_json(
-        aws;
+function (service::JSONService)(aws::AWS.AWSConfig, operation::String, args::AbstractDict{String, <:Any}=Dict{String, Any}())
+    POST_RESOURCE = "/"
+    return_headers = _return_headers(args)
+
+    request = Request(
         service=service.name,
-        version=service.api_version,
-        json_version=service.json_version,
-        target=service.target,
-        operation=operation,
-        args=args
+        api_version=service.api_version,
+        resource=POST_RESOURCE,
+        request_method="POST",
+        headers=LittleDict{String, String}(get(args, "headers", [])),
+        url=_generate_service_url(aws.region, service.name, POST_RESOURCE),
+        return_stream=get(args, "return_stream", false),
+        http_options=get(args, "http_options", []),
+        response_stream=get(args, "response_stream", nothing),
+        return_raw=get(args, "return_raw", false),
```

Comment:
```suggestion
        return_raw=get(args, "return_raw", false),
        content=json(args),
```

No reason to do it separately, right?

Reply:
No, previously we did some extra stuff with `args` but that was not needed.